app/services/process_mentions_service.rb

```diff
@@ -42,9 +42,21 @@ def call(status)
       "@#{mentioned_account.acct}"
     end
 
+    if status.limited_visibility? && status.thread&.limited_visibility?
+      # If we are replying to a local status, then we'll have the complete
+      # audience copied here, both local and remote. If we are replying
+      # to a remote status, only local audience will be copied. Then we
+      # need to send our reply to the remote author's inbox for distribution
+
+      status.thread.mentions.includes(:account).find_each do |mention|
+        status.mentions.create(silent: true, account: mention.account)
+      end
+    end
+
     status.save!
     check_for_spam(status)
 
+    # Silent mentions need to be delivered separately
```

Comment:
What happens when the status that started a conversation is deleted? No forwarding will be possible. But explicitly mentioned actors receive the reply in their inbox regardless.